### PR TITLE
Platform 366

### PR DESCRIPTION
**Pull request by jamccomb92**
### Description of proposed changes
What does this pull request accomplish? Why should it be integrated?
Script to update searchIndex for existing surveyData
Minor fix to update Object so it is not reliant on latitude/longitude
### Checklist
_Put an `x` in the boxes that apply. _
- [x] My lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my changes work (if applicable)
- [ ] I have added or updated necessary documentation (if appropriate)

### Additional comments
Is there anything else you want me to know?

### CODE REVIEW

**Comment by jamccomb92** [comment]
@hopetambala I realized that we sort of axed the search index for now, the last push (https://github.com/hopetambala/puente-node-cloudcode/pull/364/commits/89de1d6c99f50b370fbf317f1dc96a3f9dcd668a) fixed all or at least the majority of our linting issues.  We should try and add that to the repo to clean up some of those issues.

**Review by hopetambala — APPROVED** [review]

File: cloud/src/definer/crud.definer.js [hunk 1]
```diff
@@ -119,16 +119,16 @@ Parse.Cloud.define('postObjectsToClass', (request) => new Promise((resolve, reje
 
   // add key/value from the local object to SurveyPoint
   const { localObject } = request.params;
-  for (const key in localObject) {
+  localObject.forEach((key) => {
```
Comment (hopetambala) [inline cloud/src/definer/crud.definer.js:122]: @jamccomb92 you'll need to find a different way to iterate through this object, this should break

**Review by hopetambala — CHANGES_REQUESTED** [review]

File: cloud/src/definer/crud.definer.js (hunk 1 above)
Comment (hopetambala) [reply cloud/src/definer/crud.definer.js:122]: yea this unfortunately won't work

Error: {"message":"localObject.forEach is not a function","code":141}

you'll need to find a way to iterate through keys in an object

`for ... in...` is specifically for object iteration
`forEach`  is for array iteration

Comment (jamccomb92) [reply cloud/src/definer/crud.definer.js:122]: This has been fixed on most recent PLATFORM-110 PR: https://github.com/hopetambala/puente-node-cloudcode/pull/396 
This PR can be closed unless we want to continue using the searchIndex portion

**Pull request closed without merging**